examples/petstore/lib/testdata/doc/openapi.golden.json

```diff
@@ -748,8 +748,8 @@
 				]
 			},
 			"get": {
-				"description": "controller: `github.com/go-fuego/fuego/examples/petstore/controllers.PetsResources.getPets`\n\n---\n\n",
-				"operationId": "GET_/pets/:id",
+				"description": "Replace description with this sentence.",
```

Comment:
What's the reasoning behind `AddDescription` over `Description`. This looks more like it is setting it.

Reply:
No, `AddDescription` really is a string concatenation, whereas `Description` is simply setting it.